tests/lib/rules/no-deprecated-v-t.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * @author kazuya kawaguchi (a.k.a. kazupon)
+ */
+import { RuleTester } from '../eslint-compat'
+import rule from '../../../lib/rules/no-deprecated-v-t'
+import * as vueParser from 'vue-eslint-parser'
+
+const tester = new RuleTester({
+  languageOptions: {
+    parser: vueParser,
+    ecmaVersion: 2020,
+    sourceType: 'module'
+  }
+})
+
+tester.run('no-deprecated-v-t', rule as never, {
+  valid: [],
```

Comment:
Just to be sure, could you add a test case to verify that other directives are not detected?